pom.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.cloud.runtimes</groupId>
+  <artifactId>tomcat-parent</artifactId>
+  <version>0.1.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+  <name>Google Tomcat Runtime Parent</name>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <maven.build.timestamp.format>yyyy-MM-dd-HH-mm</maven.build.timestamp.format>
+    <gcloud.api.version>1.0.1</gcloud.api.version>
+    <tomcat9.minor.version>0</tomcat9.minor.version>
+    <tomcat9.dot.version>0</tomcat9.dot.version>
+    <tomcat9.version>9.${tomcat9.minor.version}.${tomcat9.dot.version}.M21</tomcat9.version>
+    <docker.tag.short>${tomcat9.minor.version}</docker.tag.short>
+    <docker.tag.long>9.${tomcat9.minor.version}-${maven.build.timestamp}</docker.tag.long>
+    <docker.openjdk.image>gcr.io/google-appengine/openjdk:8-2017-05-22_14_38</docker.openjdk.image>
+  </properties>
+
+  <modules>
+    <module>tomcat9</module>
+  </modules>
+
+  <build>
+    <pluginManagement>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>1.4.1</version>
+        <executions>
+          <execution>
+            <id>enforce-maven</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <requireMavenVersion>
+                  <version>[3.0,)</version>
+                </requireMavenVersion>
+                <requireJavaVersion>
+                  <version>[1.8,)</version>
+                </requireJavaVersion>
+              </rules>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>2.10</version>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-resources-plugin</artifactId>
```

Comment:
Do we currently need this? Also, check other plugins in this section to make sure that they all used somewhere and need versions specified.